src/php/README.md

```diff
@@ -28,10 +28,39 @@ $ curl -O http://pear.php.net/go-pear.phar
 $ sudo php -d detect_unicode=0 go-pear.phar
 ```
 
-## Build from Homebrew
+## Quick Install
 
-On Mac OS X, install [homebrew][]. On Linux, install [linuxbrew][]. Run the following command to
-install gRPC.
+**Linux (Debian):**
+
+Add [debian unstable][] (sid) to your `sources.list` file. Example:
```

Comment:
Understood. I will leave them as is for now, as there are other copy-pasta similarly in those 5 file :)